data/presets/amenity/taxi.json

```diff
@@ -4,6 +4,7 @@
         "name",
         "operator",
         "capacity",
+        "taxi_vehicle",
```

Comment:
I'd move this to the `moreFields` section: as we would have dedicated presets for the "subtypes" (e.g. _Rickshaw Stand_, …), this field would only be needed to change the "type" of an existing taxi stand (which can also be done by just changing the preset), and to display the (implicit default) value of `motorcar`.